gocourse05/HW05/main_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"testing"
+	"time"
+)
+
+func TestSaveToServer(t *testing.T) {
+	defer func() {
+		if r := recover(); r != nil {
+			t.Errorf("The code have panic")
+		}
+	}()
+
+	err := error(nil)
+	testHistory := &History{{
+		time:      time.Now(),
+		direction: right,
+		animalID:  1,
+	}}
+
+	testDayCam := DayCamera{
+		screenshot: "day_screenshot.png",
+	}
+	testNightCam := NightCamera{
+		screenshot: "night_screenshot.png",
+	}
+
+	err = testDayCam.SaveToServer("left", testHistory, 1)
```

Comment:
```suggestion
	err := testDayCam.SaveToServer("left", testHistory, 1)
```
і можна видалити `err := error(nil)` вище